context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This is for artists who know how to program, or programmers who know how to do art. Or,perhaps, for programmers who are bored.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to add simple shapes, including TITLES - words made of squares -  since these things are built in. It's also moderately easy to recursively draw a shape, as this is also a built-in function.
```

Comment:
Good catch on the TITLES in the documentation, I thought it was more difficult to do letters.
I think it would be interesting to see a recursive version of your black/grey/white squares. :-)